### DISCUSSION TITLE
What is the most detailed logger?

### DISCUSSION THREAD

**Discussion by AnkitaBarbi** [dexidp/dex#2016] [Q&A] [answered]
What is the most detailed logger?
I tried debug and info and its the same. Can someone plz help me understand is there a way to get more detailed logger?

**Comment by nabokihms** [comment, marked as answer]
Hello! The "debug" log level is the most detailed, but there is no much difference between "info" and "debug" right now.

**Reply by AnkitaBarbi** [reply]
okie, gotcha..
Is there any road map for adding more to logger?

**Reply by sagikazarmark** [reply]
Roadmap no, but we keep improving log messages as we see them.
